app/components/UnderlinedLink.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from "react";
+
+import * as styles from "./UnderlinedLink.scss";
+
+interface IUnderlinedLinkProps {
+  href: string;
+}
+
+export const UnderlinedLink: React.SFC<IUnderlinedLinkProps> = ({ href, children }) =>
```

Comment:
Some of those links might direct to outer services and then our product / design people like `_target="blank` I think this component should allow for it.

Reply:
ofc, but i would add it when someone will need it.